themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
```

Comment:
Ok, you are right, that needs more fixing. I will remove the modifications for Vanilla/base from this PR.